purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/events/CustomerCenterEventType.kt

```diff
@@ -0,0 +1,23 @@
+package com.revenuecat.purchases.customercenter.events
+
+/**
+ * Types of events that can be tracked in the Customer Center. Meant for RevenueCatUI use.
+ */
+enum class CustomerCenterEventType(val value: String) {
```

Comment:
Hmm also, does this need to be `Serializable`? Maybe we don't need the `value` property and just add `@SerialName` to each enum value so it can be serialized correctly.